packages/tasks/src/model-libraries.ts

```diff
@@ -304,6 +304,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	"mlc-llm": {
+		prettyLabel: "MLC-LLM",
+		repoName: "MLC-LLM",
+		repoUrl: "https://github.com/mlc-ai/mlc-llm",
+		filter: true,
+		countDownloads: `path_extension:"json"`,
```

Comment:
Is [mlc-chat-config.json](https://huggingface.co/mlc-ai/Llama-3-8B-Instruct-q0f16-MLC/blob/main/mlc-chat-config.json) supposed to be the only target file? If yes, let's be explicit 

```suggestion
		countDownloads: `path:"mlc-chat-config.json"`,
```

(or tokenizer.json / tokenizer_config.json ?). Better to count once when the whole repo is downloaded.

Reply:
That's a good point, I'll confirm which file should we use here.